perfkitbenchmarker/openstack/os_network.py

```diff
@@ -64,3 +66,40 @@ def AllowPort(self, vm, from_port, to_port=None, protocol='tcp'):
 
     def DisallowAllPorts(self):
         pass
+
+
+class OpenStackPublicNetwork(object):
+
+    def __init__(self, pool):
+        self.__nclient = utils.NovaClient()
+        self.__floating_ip_lock = threading.Lock()
+        self.pool = pool
```

Comment:
Could you add a comment about the lock or make the pool variable name more descriptive to match?